src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -5222,7 +5222,7 @@ public static void LengthMismatchToLower()
         }
 
         [Fact]
-        public static void ToLower()
+        public static void ToLower_2()
```

Comment:
What is the other ToLower test this is conflicting with?  "ToLower_2" isn't descriptive about what makes this test different from a presumed "ToLower_1", so we should either pick descriptive names or combine the tests.

Reply:
It's [this one](https://github.com/dotnet/runtime/blob/194376e06338bcf813c45eecb2238d339d828495/src/libraries/Common/tests/Tests/System/StringTests.cs#L5021), using `string`s + `Span` instead of `char[]` + `Span` throughout. How about naming them `ToLower_String` and `ToLower_CharArray`?

Reply:
> How about naming them ToLower_String and ToLower_CharArray?

Sounds fine. Thanks.